### PR TITLE
E2165 Fix teammate-review view

### PR DESCRIPTION
**Pull request by Arvasu**
Proposed changes can be found in the documentation link below:
- https://expertiza.csc.ncsu.edu/index.php/CSC/ECE_517_Fall_2021_-_E2165._Fix_teammate-review_view
- Contributor usernames: Anjali, Arvasu, Lalit, Vinay

### CODE REVIEW

**Comment by nnhimes** [comment]
I need to do manual testing now before it merge

File: app/views/grades/_add_icon_to_name.html.erb [hunk 1]
```diff
@@ -1,6 +1,34 @@
...
+<%# Determine the content of the reviewed by text %>
+<% review_text = "" %>
+<% if @current_role_name.eql?'Student' or questionnaire_type != "TeammateReviewQuestionnaire" %>
+  <%# Use Review {{index}} format for student view, and when questionnaire_type is not teammatereview %>
+  <% review_text = "Review #{i}" %>
+<% else %>
+  <% resp = Response.find(review.id) %>
+  <% map = ResponseMap.find(resp.map_id) %>
+  <% reviewer_id = map.reviewer_id %>
+  <% fullname = Participant.find_by(id: reviewer_id).fullname(session[:ip]) %>
```
Comment (johnbumgardner) [inline app/views/grades/_add_icon_to_name.html.erb:12]: lines 1 through 14 should really be done in whatever controller calls this

File: app/views/grades/view_team.html.erb [hunk 2]
```diff
@@ -20,6 +20,26 @@
         target.style.display = 'none';
     }
   }
+
+  function teammateSelectOnChange(selectNode) {
+    const revieweeIdToShow= selectNode.value;
```
Comment (johnbumgardner) [inline app/views/grades/view_team.html.erb:25]:
```suggestion
    const revieweeIdToShow = selectNode.value;
```

File: app/views/grades/_add_icon_to_name.html.erb [hunk 3]
```diff
@@ -1,6 +1,34 @@
+
+
+<%# Determine the content of the reviewed by text %>
+<% review_text = "" %>
+<% if @current_role_name.eql?'Student' or questionnaire_type != "TeammateReviewQuestionnaire" %>
```
Comment (johnbumgardner) [inline app/views/grades/_add_icon_to_name.html.erb:5]:
```suggestion
<% if @current_role_name.eql?'Student' || questionnaire_type != "TeammateReviewQuestionnaire" %>
```

**Review by johnbumgardner — CHANGES_REQUESTED** [review]
make changes and manual test.

**Comment by johnbumgardner** [comment]
This seems busted still.

I am going to close this Pull Request. Functionality is not here.

image

**Pull request closed without merging**